Missionframework/modules/00_init/fnc/fn_init_postInitModule.sqf

```diff
@@ -23,22 +18,29 @@
 
     Dependencies:
     NONE
+
+    Returns:
+    BOOL
 */
-if(isServer) then {
-    DEBUG_EVENTS = [
-            "KPLIB_vehicle_spawned",
-            "KPLIB_fob_built",
-            "KPLIB_sector_activated",
-            "KPLIB_sector_captured",
-            "KPLIB_sector_deactivated",
-            "KPLIB_player_fob"
-        ];
-
-    {
-        [_x, compile ("diag_log 'eventCalled " + _x + "'; diag_log _this;")] call CBA_fnc_addEventHandler;
-        [_x, compile ("systemChat 'eventCalled " + _x + "'; systemChat format['%1', _this];")] call CBA_fnc_addEventHandler;
-    } forEach DEBUG_EVENTS;
-};
+
+if (!isServer) exitWith {};
+
+diag_log format ["[KP LIBERATION] [%1] [POST INIT] Module initializing...", diag_tickTime];
+
+// Initialize Debug Events
+DEBUG_EVENTS = [
```

Comment:
It seems I've added this by accident in 3d1a84d33221ce3ca7431c3f9eb2d770d0d108a5. 😅 
 Can be safely removed.

Reply:
done in https://github.com/KillahPotatoes/KP-Liberation/pull/483/commits/ae19ef9c7736f3c03ec858492562e38ebd839446